docs/contents.md

```diff
@@ -14,134 +14,143 @@ The **FortiAI** solution pack contains the following resources:
                             </tr>
                         </table>
                     </td> 
                 </tr>
                 <tr> 
-                    <th>fieldsOfInterest</th>
+                    <th><code>fieldsOfInterest</code></th>
                     <td>To include the field data for the that module in conversation.</td>
                 </tr>
             </table>
         </td>
     </tr>    
 </table>
 
-<table>
-    <tr>
-        <th>NOTE</th>
-    </tr>
-    <tr>
-        <td>
-            <ol>
-                <li>To add the new LLM or LLM Models, refer <a href="./llm-configuration.json">JSON</a>.</li>
-                <li>To add the new question, refer <a href="./question-sample.json">JSON</a>.</li>
-            </ol>
-        </td>
-    </tr>
-</table>
+>[!NOTE]
+>To add a deprecated question for different modules, follow these steps:
+>   1. Add the question in JSON format under the particular module's API name. A sample [Question JSON](./question-sample.json) is attached to the document.
+>   2. Add new LLM or LLM models in [](./),Write a playbook along with the tag mentioned in the JSON above.
 
 ## Playbook Collection
 
-|10 - SP - Fortinet Advisor |
-|:-------------------------:|
-
-
-| Playbook Name                                             | Description                                                                       |
-|:----------------------------------------------------------|:----------------------------------------------------------------------------------|
-| Get Playbook Step Suggestion                              | Suggests steps to create a playbook based on user's query.                        |
-| Get Playbook Step Suggestion (Loop)                       | Suggests steps to create a playbook based on user's query, in a loop.             |
-| Get Playbook Block Suggestion                             | Creates a suggested playbook block to perform actions based on user requirements. |
-| > Get LLM Response                                        | Reference playbook for getting a response from the LLM.                           |
-| > Get LLM Configuration | Get the LLM configuration based on the value defined for the key "isMultiConfigAvailable" |
-
->**Warning:** We recommend that you clone these playbooks before customizing to avoid loss of information while upgrading the solution pack.
+|10 - SP - FortiAI |
+|:----------------:|
+
+
+| Playbook Name                                                                  | Description                                                                               |
+|:-------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------|
+| Get Playbook Step Suggestion                                                   | Suggests steps to create a playbook based on user's query.                                |
```

Comment:
Many of these playbooks are removed. Please verify once and make required changes.

Reply:
@cs-ankit-gupta  for your notice

Reply:
Marked deprecated